### PR TITLE
update requirements in Stufe 2 according to Stufe 3 und 4

### PR DESCRIPTION
**Pull request by f-peverali**
# Contributor Pull Request

## Description
Suchparameter für die Slot Elemente .schedule , .start und .status wurden auf die Anforderung aus Stufe 3 und 4 nachträglich reduziert.

## Motivation and Context

Implementierungsaufwand reduzieren, da in Folgestufen nicht notwendig.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this IG / specification.
- [ ] My change requires a change to the documentation or narrative (intend) of the IG.
- [x] I have already updated the documentation / narrative (intend) accordingly.

### CODE REVIEW

File: ImplementationGuide/markdown/Datenobjekte/ISiKTerminblockSlot.md [hunk 1]
```diff
@@ -73,27 +73,31 @@ Beim Fehlen des "start"-Suchparameters SOLL der aktuelle Zeitpunkt des Servers a
 
     Beispiele:
 
-    ```GET [base]/Slot?schedule=Schedule/ISiKKalenderExample:start=2022-12-10T09:00:00Z```
+    ```GET [base]/Slot?schedule=Schedule/ISiKKalenderExample&&start=2022-12-10T09:00:00Z```
```
Comment (alexey-tschudnowsky) [inline ImplementationGuide/markdown/Datenobjekte/ISiKTerminblockSlot.md]: type: &&

Comment (f-peverali) [inline ImplementationGuide/markdown/Datenobjekte/ISiKTerminblockSlot.md]:
```suggestion
    ```GET [base]/Slot?schedule=Schedule/ISiKKalenderExample&start=2022-12-10T09:00:00Z```
```

File: ImplementationGuide/markdown/ReleaseNotes.md [hunk 2]
```diff
@@ -6,6 +6,14 @@ Die erste Ziffer X bezeichnet ein Major-Release und regelt die Gültigkeit von R
 
 ----
 
+Version 2.0.6
+
+Datum: tbd.
+
+* Ernuete Änderung der Anforderung für Suchanfrage zu Terminblöcken aus Kalendern im Sinne einer Verfeinerung der Anforderung und Anpassung an Stufe 3 und 4
```
Comment (f-peverali) [inline ImplementationGuide/markdown/ReleaseNotes.md]:
```suggestion
* Erneute Änderung der Anforderung für Suchanfrage zu Terminblöcken aus Kalendern im Sinne einer Verfeinerung der Anforderung und Anpassung an Stufe 3 und 4 
```

File: ImplementationGuide/markdown/Datenobjekte/ISiKTerminblockSlot.md [hunk 3]
```diff
@@ -73,27 +73,31 @@ Beim Fehlen des "start"-Suchparameters SOLL der aktuelle Zeitpunkt des Servers a
...
 
     ```GET [base]/Slot?schedule=Schedule/ISiKKalenderExample```
 
     Anwendungshinweise: Weitere Informationen zur Suche nach "Slot.schedule" finden sich in der [FHIR-Basisspezifikation - Abschnitt "reference"]https://hl7.org/fhir/R4/search.html#reference).
-    
+
     In diesem Fall ist auch ein Chaining auf weitere verknüpfte Akteure möglich: `GET https://example.org/fhir/Slot?schedule.actor:HealthcareService.type=https://example.org/fhir/CodeSystem/Behandlungsleistung|CT`
 
-3. Der Suchparameter "status" MUSS unterstützt werden:
+    Dies kann notwendig sein, falls interdisziplinäre Kalender durch das Termin-Repository gepflegt werden (z.B. ein Kalender pro Station). In solchen Fälle wäre es sinnvoll, wenn das Termin-Repository einem Termin-Requestor eine Verknüpfung zwischen Kalender (Schedule) und Behandlungsleistung (HealthcareService) bereitstellen würde.
```
Comment (f-peverali) [inline ImplementationGuide/markdown/Datenobjekte/ISiKTerminblockSlot.md]:
```suggestion
    Dies kann notwendig sein, falls interdisziplinäre Kalender durch das Termin-Repository gepflegt werden (z.B. ein Kalender pro Station). In solchen Fällen wäre es sinnvoll, wenn das Termin-Repository einem Termin-Requestor eine Verknüpfung zwischen Kalender (Schedule) und Behandlungsleistung (HealthcareService) bereitstellen würde.
```

**Pull request merged**